script.js

```diff
@@ -1,10 +1,10 @@
-// function closeMenu() {
-//   document.getElementById("menu").style.display = "none";
-// }
+function closing() {
+  document.getElementById('menu').style.display = 'none';
+  window.location.reload();
+}
```

Comment:
Hi @valt19, good job 👍 , but there are still some issues you need to fix to get your work approved!

As requested by the previous reviewer and according to the [project requirements ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md)
- When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

When I clicked on any of the mobile menu options, the correct part of the page is displayed but the mobile menu does not disappear.

when I scrolled to the top of the page.  I can still find the mobile menu displayed 👇 

![image](https://user-images.githubusercontent.com/78251927/226093127-a3521f03-8c65-47fd-bdfb-5e1bf849d0e0.png)